common.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../resources/project.rnc" type="application/relax-ng-compact-syntax"?>
+<project xmlns="https://www.dita-ot.org/project">
+  <context id="html">
+    <input href="userguide.ditamap"/>
```

Comment:
In contrast to the JSON and YAML variants, this file has no `profiles`/`ditavals`, so the generated HTML output currently differs depending on which project file format is used.

We should probably make them equivalent, so the files just illustrate how to achieve the same result with different formats.

Reply:
Resolved in 762fa6b.